contracts/USM.sol

```diff
@@ -131,4 +148,75 @@ contract USM is BufferedToken {
         latestFumPrice = fumPrice();
         emit LatestFumPriceChanged(previous, latestFumPrice);
     }
+
+    /** INTERNAL FUNCTIONS */
+
+    /**
+     * @notice Funds the pool with ETH, minting FUM at its current price
+     */
+    function _fund(address to, uint ethIn) internal {
+        uint _fumPrice = fumPrice();
+        uint fumOut = ethIn.wadDiv(_fumPrice);
+        ethPool = ethPool.add(ethIn);
+        fum.mint(to, fumOut);
+        _setLatestFumPrice();
+    }
+
+    /**
+     * @notice Mint USM from Eth. Inheriting contract must handle the Ether transfers.
+     */
+    function _mint(uint ethAmount) internal returns (uint) {
+        uint usmAmount = _ethToUsm(ethAmount);
+        ethPool = ethPool.add(ethAmount);
+        super._mint(msg.sender, usmAmount);
+        return usmAmount;
+    }
+
+    /**
+     * @notice Burn USM for Eth. Inheriting contract must handle the Ether transfers.
+     */
+    function _burn(uint usmAmount) internal returns (uint) {
+        uint ethAmount = _usmToEth(usmAmount);
+        ethPool = ethPool.sub(ethAmount);
+        super._burn(msg.sender, usmAmount);
+        return ethAmount;
+    }
+
+    /**
+     * @notice Convert ETH amount to USM using the latest price of USM
+     * in ETH.
+     *
+     * @param _ethAmount The amount of ETH to convert.
+     * @return The amount of USM.
+     */
+    function _ethToUsm(uint _ethAmount) internal view returns (uint) {
```

Comment:
I would make this one public, it should be useful for users.

Reply:
One thing to keep in mind here is that once we implement sliding prices, `ethToUsm()` becomes more complex...  Because suddenly the conversion from ETH to USM not only isn't linear any longer, but changes depending whether you're `minting`, `burning`, etc.  But we'll cross this bridge when we come to it.